tests/manager_caller.c

```diff
@@ -0,0 +1,62 @@
+#include "manager.h"
+
+extern struct Compartment* loaded_comp;
+
+struct CompEntryPoints
+{
+    char* file_name;
+    char** entry_points;
+    char** args;
+};
+
+// TODO
```

Comment:
TODO?

Reply:
This is to remind myself to remove the `CompEntryPoints` code once I transition to defining this in compartment configuration files (after this PR).